tools/c7n_tencentcloud/c7n_tencentcloud/resources/vpc.py

```diff
@@ -0,0 +1,69 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.utils import type_schema, local_session
+
+from c7n.filters import Filter
+
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod
+
+
+@resources.register("vpc")
+class VPC(QueryResourceManager):
+    """
+    vpc - Virtual Private Cloud (VPC)
+    https://www.tencentcloud.com/document/product/215/535?lang=en&pg=
+    """
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "VpcId"
+        endpoint = "vpc.tencentcloudapi.com"
+        service = "vpc"
+        version = "2017-03-12"
+        enum_spec = ("DescribeVpcs", "Response.VpcSet[]", {})
+        metrics_instance_id_name = "natId"
+        paging_def = {"method": PageMethod.Offset, "limit": {"key": "Limit", "value": "20"}}
+        resource_prefix = "vpc"
+        taggable = True
+
+
+@VPC.filter_registry.register('flow-logs')
+class FlowLogFilter(Filter):
+    """Are flow logs enabled on the resource.
+
+    ie to find all vpcs with flows logs disabled we can do this
+
+    :example:
+    .. code-block:: yaml
+            policies:
+              - name: flow-mis-configured
+                resource: vpc
+                    filters:
+                      - type: flow-logs
+                        enabled: false
```

Comment:
same comment as per lifecycle-rule, users want to be able to filter on attributes of the flow log
https://www.tencentcloud.com/document/product/215/36908

Reply:
same comment as per lifecycle-rule,  use ValueFilter to filter flow log attributes